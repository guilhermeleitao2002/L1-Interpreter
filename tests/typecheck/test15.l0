type Person = struct{ #name:int, #age: int };
let p0:Person = {#name = 1};
p0.#name + p0.#age;;