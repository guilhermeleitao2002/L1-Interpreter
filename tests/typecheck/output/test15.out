TypeError: Expression type struct { name: int } does not match declared type struct { name: int, age: int }